Uplift/Views/ClassList/CalendarCell.swift

```diff
@@ -64,14 +64,14 @@ class CalendarCell: UICollectionViewCell {
         dateLabelCircle.layer.cornerRadius = 12
         addSubview(dateLabelCircle)
 
-        dateLabel.text = "15"
+        dateLabel.text = "15" //Placeholder
```

Comment:
do we need all these placeholders? can they just get set via the `configure`?

Reply:
You're right, they make no difference so I removed the ones I found.